lib/mongo/crypt/binding.rb

```diff
@@ -793,7 +793,7 @@ def self.kms_ctx_message(kms_context)
           mongocrypt_kms_ctx_message(kms_context.kms_ctx_p, binary.ref)
         end
 
-        return binary.to_string
+        return binary.to_s
```

Comment:
Can this fix be in a separate PR please with the ticket reference to the to_string -> to_s change?